static/app/views/alerts/wizard/index.tsx

```diff
@@ -34,47 +34,31 @@ type RouteParams = {
   projectId?: string;
 };
 
-type Props = RouteComponentProps<RouteParams, {}> & {
+type AlertWizardProps = RouteComponentProps<RouteParams, {}> & {
   organization: Organization;
   projectId: string;
 };
 
-type State = {
-  alertOption: AlertType;
-};
-
 const DEFAULT_ALERT_OPTION = 'issues';
 
-class AlertWizard extends Component<Props, State> {
-  state: State = {
-    alertOption:
-      this.props.location.query.alert_option in AlertWizardAlertNames
-        ? this.props.location.query.alert_option
-        : DEFAULT_ALERT_OPTION,
-  };
-
-  componentDidMount() {
-    // capture landing on the alert wizard page and viewing the issue alert by default
-    this.trackView();
-  }
-
-  trackView(alertType: AlertType = DEFAULT_ALERT_OPTION) {
-    const {organization} = this.props;
-    trackAnalytics('alert_wizard.option_viewed', {
-      organization,
-      alert_type: alertType,
-    });
-  }
-
-  handleChangeAlertOption = (alertOption: AlertType) => {
-    this.setState({alertOption});
-    this.trackView(alertOption);
+function AlertWizard({
+  organization,
+  params,
+  location,
+  projectId: _projectId,
```

Comment:
Why does it need renamed?

Reply:
oh before it was because we had two of them. We attempt to get the project id from two places

Reply:
wait so do we still need this?

Reply:
yeah